### ISSUE TITLE
Strict Concurrency / Swift 6 Compatibility

### ISSUE THREAD

**Issue by Brett-Best** [ikhvorost/ReactBridge#5]
Just wondering if you've had any thought on ways to address the strict concurrency warnings which will become errors in Swift 6?

**Comment by ikhvorost** [comment]
Hi @Brett-Best! 

Do you mean these warnings?

```
warning: static property 'methodInfo' is not concurrency-safe because it is non-isolated global shared mutable state; this is an error in Swift 6
warning: reference to static property 'methodInfo' is not concurrency-safe because it involves shared mutable state; this is an error in Swift 6
```

**Comment by Brett-Best** [comment]
Yep, so far those the only ones I've ran into

**Comment by ikhvorost** [comment]
It'll be fixed in new release. Thanks.

**Comment by ikhvorost** [comment]
Warnings are fixed in [1.1.0](https://github.com/ikhvorost/ReactBridge/releases/tag/1.1.0). Thanks.

**Comment by Brett-Best** [comment]
Thanks 💜
